src/vstart.sh

```diff
@@ -126,9 +126,14 @@ usage=$usage"\t-X disable cephx\n"
 usage=$usage"\t--hitset <pool> <hit_set_type>: enable hitset tracking\n"
 usage=$usage"\t-e : create an erasure pool\n";
 usage=$usage"\t-o config\t\t add extra config parameters to all sections\n"
+usage=$usage"\tmon : start just ceph monitor's\n"
+usage=$usage"\tosd : start just ceph osd's\n"
+usage=$usage"\tmds : start just ceph mds's\n"
```

Comment:
the plurals shouldn't have `'` in them (and the plural of 'mds' is 'mdses' according to @adamemerson)